example10/submitter_by_id.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import flux
+import sys
+import os
+
+from flux import job
+from flux.job import JobspecV1
+
+if len(sys.argv) != 2:
+    njobs = 10
+else:
+    njobs = int(sys.argv[1])
+
+# open connection to broker
+h = flux.Flux()
+
+# create jobspec for compute.py
+compute_jobspec = JobspecV1.from_command(
+    command=["./compute.py", "30"], num_tasks=4, num_nodes=2, cores_per_task=2
```

Comment:
```suggestion
    command=["./compute.py", "10"], num_tasks=4, num_nodes=2, cores_per_task=2
```

For the purposes of interactivity, I think shrinking this window would be a good idea.